### PR TITLE
Ported Crawlera User Guide to rst

### PR DESCRIPTION
**Pull request by qrilka**
@giopiazza would you please check design of this update?
@rdowinton I have removed restricted headers, please take a look if I haven't missed somethig

### CODE REVIEW

**Comment by giopiazza** [comment]
Looks nice! Good work @qrilka :smile: 

I just committed a minor change on CSS (it was a minor adjustment overall, not specific of crawlera page, but it will also contribute to its layout). Ok?

In my humble opinion is good to go :+1: 

@rdowinton what do you think?

**Comment by qrilka** [comment]
thanks @giopiazza 
@rdowinton I think we probably need to prepare some announcement about Crawlera section
@ittailup what do you think?

**Comment by rdowinton** [comment]
Looks good @qrilka @giopiazza !

Yeah, we should announce the section so people can link it rather than attach PDF.  @serhiy-yasko  mentioned updating the signup template with link to PDF (via S3), so we probably should update that too.

**Comment by qrilka** [comment]
I was rather referring to some public announcement  ( @ittailup was saying about preparing some list of Crawlera customer emails)

File: crawlera.rst [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+Note that it’s not recommended to use the fetch API with the Crawlera middleware. Another option is the X-Crawlera-Use-HTTPS header.
+
+
+Errors
+======
+
+When an error occurs, Crawlera will send a response containing an X-Crawlera-Error header and an error message in the body.
+
+Warning: These errors are internal to Crawlera and are subject to change at any time, so it’s strongly recommended you avoid relying on them for anything important:
```
Comment (pablohoffman) [inline crawlera.rst]: I'd reword to "strongly recommended you use them only for debugging and avoid relying on them"

"anything important" is too subjective :)
